docs/api/telegram.rst

```diff
@@ -372,6 +372,11 @@ about its business.
 
       .. versionadded:: 0.3
 
+   .. py:method:: delete_message(message)
+
+      Delete the selected message.
+      A message can be deleted only if is sent by the bot or sent in a supergroup by an user where the bot is admin.
+
```

Comment:
You forgot to add the versionadded (0.4) and the documentation for the parameter.